src/pl/senla/hotel/service/ServiceFacilityImpl.java

```diff
@@ -63,7 +63,7 @@ public HotelFacility read(int idFacility) {
 
     @Override
     public boolean update(int idFacility, String hotelFacilityString) {
-        if(repositoryHotelFacility.readAll() == null || repositoryHotelFacility.readAll().isEmpty()){
+        if(daoHotelFacility.readAll() == null || daoHotelFacility.readAll().isEmpty()){
```

Comment:
why do you read all?? you need only update hotelFacilityString by id (hotelFacilityString - not self-describing, change it), there is only one check you can do here is a check entity in DB that exists by id